tests/e2e/main.go

```diff
@@ -208,6 +200,10 @@ func parseArguments() (err error) {
 
 	flag.Var(&selectedTestfiles, "test-file",
 		getTestFileUsageString())
+
+	flag.Var(&consumerVersions, "cv", "Consumer version")
```

Comment:
could you document the default behaviour for these and make the help strings more descriptive?